radicle/src/cob/issue.rs

```diff
@@ -387,7 +416,7 @@ impl<'a> Issues<'a> {
 #[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize)]
 #[serde(tag = "type", rename_all = "camelCase")]
 pub enum Action {
-    Title { title: String },
+    Edit { title: String },
```

Comment:
I explained it in one of the last commits, but it's to be consistent with the `Patch` actions, where you can edit a couple of other things as well.

The way I've been thinking about it is that the `Edit` action edits the issue metadata, which for now is only the title, but in the future could be other things that are not worth having a dedicated action for.

Comment editing would go under the `Thread` action, since the comments are part of the Thread CRDT.

Does that make sense?

Reply:
> I explained it in one of the last commits, but it's to be consistent with the Patch actions, where you can edit a couple of other things as well.

Ah cool, sorry that I missed that :) 

> Does that make sense?

Yup, thanks for the explanation!